### PR TITLE
Fixed shellcheck warnings in setup scripts

### PR DESCRIPTION
**Pull request by MathewMacDougall**
### Description

Noticed some [shellcheck](https://www.shellcheck.net/) warnings in our setup scripts while I was removing arch stuff. Decided to quickly fix most of them to appease my vim plugin.

### Testing Done

Ran some locally (cube, openocd)
CI

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

**Review by jonathanlew — APPROVED** [review]
Didn't know there was a vim plugin that checked that. LGTM pending CI

File: environment_setup/install_openocd.sh [hunk 1]
```diff
@@ -32,4 +32,6 @@ sudo make install
 
 echo "================================================================"
 echo "Done Installing OpenOCD"
+echo ""
+echo "Make sure to logout or restart your computer for permissions changes to take effect!"
```
Comment (akhilveeraghanta) [inline environment_setup/install_openocd.sh:36]: should we do `sudo systemctl restart udev` or is that too invasive? I think that would avoid having to restart

File: environment_setup/setup_udev_rules.sh [hunk 2]
```diff
@@ -17,25 +17,25 @@ udev_rules_file="/etc/udev/rules.d/99-thunderbots.rules"
...
 # This allows the udev rules file to properly grant USB permission
 sudo groupadd thunderbots
-sudo adduser $USER thunderbots
+sudo adduser "$USER" thunderbots
 
 echo "================================================================"
 echo "Done"
 echo ""
-echo "Make sure to restart your computer for permissions changes to take effect!"
+echo "Make sure to logout or restart your computer for permissions changes to take effect!"
```
Comment (akhilveeraghanta) [inline environment_setup/setup_udev_rules.sh:39]: ditto previous comment, might we worth just restarting udev daemon

**Review by akhilveeraghanta — APPROVED** [review]
lgtm, only got one nonblocking comment

File: environment_setup/install_openocd.sh (hunk 1 above)
Comment (MathewMacDougall) [reply environment_setup/install_openocd.sh:36]: Oh I didn't know you could do that. @garethellis0 @EvanMorcom @jonathanlew thoughts?

Comment (garethellis0) [reply environment_setup/install_openocd.sh:36]: I'm fine with that, let's just make sure to test it locally first.

**Review by garethellis0 — APPROVED** [review]

**Comment by MathewMacDougall** [comment]
Merging for now, we can revisit the udev restart separately

**Pull request merged**